src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="fix">
```

Comment:
add more meaningful class names
```suggestion
      <a href="index.html" class="logo_link">
```

Reply:
Not fixed